src/components/BottomSheetInLineNotification.tsx

```diff
@@ -17,9 +18,21 @@ interface Props extends InLineNotificationProps {
 const NOTIFICATION_HEIGHT = 500
 
 // for now, this notification is launched from the bottom of the screen only
-const BottomSheetInLineNotification = ({ showNotification, ...inLineNotificationProps }: Props) => {
+const BottomSheetInLineNotification = ({
+  showNotification,
+  onUnmount,
+  ...inLineNotificationProps
+}: Props) => {
   const [isVisible, setIsVisible] = useState(showNotification)
 
+  useEffect(() => {
```

Comment:
This unmount hook is needed for the edge case where the user does not dismiss the error banner, and opens the side nav to navigate away from the page. When the banner is visible, the settings screen is otherwise uninteractable, but the user can still open the side nav. Without this hook, the user would see the error banner immediately the next time they navigate to the settings screen.